### ISSUE TITLE
`btcli stake child set` stopped working after subtensor 8.5.1 upgrade

### ISSUE THREAD

**Issue by antonr-p2p** [opentensor/btcli#276]
`btcli stake child set` stopped working after subtensor 8.5.1 upgrade.
It performs all the operations without errors, but nothing changes. 
Even output after execution is the same as it was before.
It looks like it's related to the subtensor 8.5.1 upgrade.

Env tested:
CLI version 8.4.2

**Comment by antonr-p2p** [comment]
@ibraheem-opentensor @thewhaleking sorry for pinging you directly, unfortunately that's a pretty urgent issue. Because of CR3 enablement some validators are stuck/has low vTrust and there is an urgent need to redelegate validator's stake for many SNs.

**Comment by ibraheem-opentensor** [comment]
@antonr-p2p Can you confirm that the childkeys get set after 7200 blocks? 

There was a recent change which introduces a cooldown period. If you try again within that period, the cooldown will reset. 

Please see more detailed explanation here: 

https://discord.com/channels/799672011265015819/799672011814862902/1319417265707286569

Let me know if this answers your query.

**Comment by antonr-p2p** [comment]
@ibraheem-opentensor yep, I confirm after 1 day delay it works. Sorry for pinging you, I was confused.

BTW can we improve btcli to show proper info about this? I think it's good to do like this:
1. For `btcli stake child set` after finilizing show warning that it would be applied with delay of 1 day
2. For `btcli stake child get` if change is in progress - show current and proposed states with time left (if possible)

This change would help others and would stop them from creating such false positive issues.
